modules/hcp-ts-ingress-not-online.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ts-ingress-not-online_{context}"]
+= Ingress and console cluster operators are not coming online
+
+If a hosted control plane is not coming fully online because the Ingress and console cluster operators are not online, check the wildcard DNS routes and load balancer.
```

Comment:
```suggestion
If a hosted control plane is not coming fully online because the Ingress and console cluster Operators are not online, check the wildcard DNS routes and load balancer.
```